client/extensions/zoninator/components/settings/zone/post-card.jsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import SectionHeader from 'components/section-header';
+
+class PostCard extends Component {
+
+	handleRemove = () => {
+		this.props.remove( this.props.cardIdx );
+	}
+
+	handleMouseDown = ( event ) => {
+		event.stopPropagation();
+		event.preventDefault();
+	}
+
+	render() {
+		const {
+			translate,
+			post: {
+				ID,
+				URL,
+				site_ID,
+				title,
+				slug,
+			}
+		} = this.props;
+
+		const postCardClass = 'zoninator__zone__list-item';
+
+		return (
+			<SectionHeader key={ slug } label={ title } className={ postCardClass }>
+				<Button
+					compact
+					onMouseDown={ this.handleMouseDown }
+					href={ URL }
+					target="_blank"
+					rel="noopener noreferrer">
+					{ translate( 'View' ) }
+				</Button>
+				<Button
+					compact
+					onMouseDown={ this.handleMouseDown }
+					href={ `/post/${ site_ID }/${ ID }` }
+					target="_blank"
```

Comment:
Sure we want to open this in a new tab? I think we try to stay inside the same tab as much as possible, if the target is also inside Calypso, even in cases like this. Easy enough for the user to hit 'Back'.

Reply:
(Or to explicitly open it in a new tab, if they want to.)